src/exportImage.py

```diff
@@ -15,6 +15,16 @@ def get_zoom_resolution(zoom_to_resolution_dict, zoom_level):
         raise Exception(f'No such zoom level. got: {zoom_level}')
 
 
+def is_valid_zoom_for_area(resolution, tile_size, bbox):
```

Comment:
what is the need in the tile_size?

Reply:
Currently, a tile is 256x256, so 256 is the size.